VariablesHomework/VariablesHomework.cs

```diff
@@ -161,18 +162,18 @@ public void Casting()
             // what am i doing here that make the above work?
             double d = 15.5;
             int c = (int)d;
-            Assert.IsTrue(c == /*Answer here*/); // Make this pass
+            Assert.IsTrue(c == (int)d); // Make this pass
```

Comment:
Again, I'm looking for a literal value

Reply:
Changed to (c == (int)15.5);
Changed to (e == "15.5");